pkg/task/backup.go

```diff
@@ -195,21 +195,24 @@ func RunBackup(c context.Context, g glue.Glue, cmdName string, cfg *BackupConfig
 	if err != nil {
 		return err
 	}
-
-	if cfg.LastBackupTS == 0 {
-		var valid bool
-		valid, err = client.FastChecksum()
+	checksums, err := client.FastChecksum()
```

Comment:
It seems we still have checksum progress bar here when `cfg.Checksum`=false

Reply:
That was done... Even a little complex, but I think that is acceptable.

Reply:
okay，I think we can move L207-L226 to L184-L198, it will looks like
```
if cfg.LastBackupTS == 0 {
...
return
}

if cfg.Checksum {
...
} else {
...
}

Reply:
OK, that was done.